client/src/components/App/AppToolbar/AppToolbar.tsx

```diff
@@ -47,6 +47,8 @@ const AppToolbar: React.FC = (): JSX.Element => {
 
   const classes = useStyles();
   const location = useLocation();
+  const userName = user.authorityByAuthorityId?.authorityName ? 
```

Comment:
I'm not sure the userName is only the user name, in case it is you can leave it with the name and export -
user.userName +" (" + user.authorityByAuthorityId.authorityName + ")"
to another const
if it's not only the user name maybe change the name to something else that will be more clear